### DISCUSSION TITLE
请问各位，Bilibili Evolved的这些功能如何使用？

### DISCUSSION THREAD

**Discussion by Yukihana-Lamy** [the1812/Bilibili-Evolved#4505] [提问 / Q&A] [answered]
请问各位，Bilibili Evolved的这些功能如何使用？

![Bilibili Evolved功能1](https://github.com/the1812/Bilibili-Evolved/assets/103800975/abf926a8-22a9-433e-9ae9-56ba6273ec4c)

**Comment by Geeyun-JY3** [comment, marked as answer]
组件各自的简介描述得还不够清楚吗：https://github.com/the1812/Bilibili-Evolved/blob/master/doc/features/features.md
